README.md

```diff
@@ -70,14 +81,42 @@ class _HomeState extends State<Home> {
   @override
   void initState() {
     super.initState();
+    checkVersion();
+  }
 
-    Future.delayed(Duration.zero, () {
-      NativeUpdater.displayUpdateAlert(
-        context: context,
-        requiresUpdate: true,
-        appStoreUrl: '<Your App Store URL>',
-      );
-    });
+  Future<void> checkVersion() async {
+    /// For example:
+    /// You've got status code of 412 from the response of HTTP request.
+    /// Let's say the statusCode 412 is requires you to force update
+    /// and statusCode 200 ik OK but new version available.
+
+    int statusCode = 412; /// Try by switching the value to 412 or 200
+
+    if (statusCode == 412) {
+      Future.delayed(Duration.zero, () {
+        NativeUpdater.displayUpdateAlert(
+          context: context,
+          forceUpdate: true,
+          // appStoreUrl: '<Your App Store URL>',
```

Comment:
Please remove comments and keep all texts in English.

Reply:
I'm on it...

Reply:
Resolved in #5